src/main/java/gui/pages/HomePagePartners.java

```diff
@@ -0,0 +1,12 @@
+package gui.pages;
+
+import com.codeborne.selenide.Selenide;
+import gui.BasePage;
+
+public class HomePagePartners extends BasePage {
+
+    public HomePagePartners() {
+        // Wait for page to load
+        Selenide.Wait().until(Selenide::page);
```

Comment:
You are repeating this method. Should be initialized in BasePage if it will be used more than once